include/SFML/Graphics/Shader.hpp

```diff
@@ -58,8 +58,9 @@ class SFML_GRAPHICS_API Shader : GlResource, NonCopyable
     ////////////////////////////////////////////////////////////
     enum Type
     {
-        Vertex,  ///< Vertex shader
-        Fragment ///< Fragment (pixel) shader
+        Vertex =   1 << 0, ///< Vertex shader
+        Geometry = 1 << 1, ///< Geometry shader
+        Fragment = 1 << 2  ///< Fragment (pixel) shader
```

Comment:
Yeah this must have been a leftover. The original code went through a rewrite, and that went through a rewrite again, and that again... maybe like 4 times over the last year. :stuck_out_tongue:

Will fix.